extensions/data-workspace/src/dialogs/dialogBase.ts

```diff
@@ -18,8 +20,9 @@ export abstract class DialogBase {
 	protected _dialogObject: azdata.window.Dialog;
 	protected initDialogComplete: Deferred<void> | undefined;
 	protected initDialogPromise: Promise<void> = new Promise<void>((resolve, reject) => this.initDialogComplete = { resolve, reject });
-	protected workspaceFormComponent: azdata.FormComponent | undefined;
-	protected workspaceInputBox: azdata.InputBoxComponent | undefined;
+	protected workspaceDescriptionFormComponent: azdata.FormComponent | undefined;
```

Comment:
😄 this base class is meant to be a generic base class for any dialogs, but now it is a base class only for project dialogs, it is ok for now, but when we need to introduce other dialogs we need to split this class.